tree/ntuple/v7/inc/ROOT/libdaos_mock/daos.h

```diff
@@ -238,13 +243,78 @@ int daos_obj_update(daos_handle_t oh, daos_handle_t th, uint64_t flags,
 
 
 //////////////////////////////////////////////////////////////////////////////// daos_prop.h
-
+#include <ctype.h>
```

Comment:
Why is this needed?  Maybe move to the beginning of the file together with other `#include`s.

Reply:
This is due to `isalnum` and `isxdigit` in `daos_label_is_valid` from `daos_prop.h`, which might be removed altogether (or simplified if used for development)